src/Grid/Col/index.js

```diff
@@ -54,4 +55,5 @@ Col.defaultProps = {
   md: null,
   sm: null,
   xs: null,
+  center: null,
```

Comment:
are these unrelated changes?

Reply:
this one was actually cause I have changed the `SideBar.jsx` to `SideBar.tsx` and there was a prop in the Col which is this one but not mentioned in the types so I added it here cause the next ticket is about changing sidebar to `tsx` in both story and component

Reply:
Ok so lets remove this change from this PR as its unrelated, you can add it back to the sidebar PR. Otherwise its difficult to review one chunk or work and say its working and makes sense as sits split across multiple PRs with reasonings existing for other unknown reasons

Reply:
already done in the last commits 😃

Reply:
ok, trying it